cpp/src/toppra/solver/seidel.cpp

```diff
@@ -195,31 +77,59 @@ LpSol solve_lp2d(const RowVector2& v,
   // adhered to: fixed bounds are assigned the numbers: -1, -2, -3,
   // -4 according to the following order: low[0], high[0], low[1],
   // high[1].
-  if ((low.array() > high.array()).any()) {
-    TOPPRA_LOG_DEBUG("Seidel: incoherent bounds.");
-    return INFEASIBLE;
-  }
   for (int i = 0; i < 2; ++i) {
+    if (low[i] > high[i]) {
```

Comment:
I am not sure I understand the purpose of this section. If there is an i such that low[i] and high[i], shouldn't the problem be declared infeasible?